src/components/Transfer.tsx

```diff
@@ -42,24 +43,31 @@ const useStyles = makeStyles((theme) => ({
   }
 }));
 
-const Transfer = () => {
+function Transfer() {
   const classes = useStyles();
   const [isRunning, setIsRunning] = useState(false);
+  const [helperText, setHelperText] = useState('');
   const [transferInput, setTransferInput] = useState<string>('');
+  const [actualInput, setActualInput] = useState<number | null>();
   const { sourceChainDetails, targetChainDetails } = useSourceTarget();
 
   const areApiReady = useLoadingApi();
-
+  const transformToBase = 10 ** targetChainDetails.targetApiConnection.api.registry.chainDecimals[0];
```

Comment:
So it kind of is `base` already, we transform to smallest derived units. For lack of a better word, maybe we can call them `planck` units? :)